tracer/test/test-applications/integrations/Samples.InstrumentedTests/Vulnerabilities/SqlInjection/EFTests.cs

```diff
@@ -51,5 +52,21 @@ protected override EntityCommand GetEntityCommand(string title)
         var query = new EntityCommand(queryString, conn);
         return query;
     }
+
+    [Fact]
+    public void TestDelegateDecompileLibBug()
+    {
+        var book = new Book { Id = "id", Title = "title", Author = "author" };
+        var ft = book.FullTitle;
+        var books = (db as ApplicationDbContext).Books;
+        var decompiled = books.Decompile();
+        var any = decompiled.Any(x => x.FullTitle != ft);
+        //var data = (db as ApplicationDbContext).Books.Decompile().Where(x => x.FullTitle != "eeef").ToList();
```

Comment:
I would remove the commented code, or at least explain why it's there.

Reply:
Fixed in 8a6bab0e64c2b530becf1801385ca75cabc4c82a